_Week8/Group4/simulate.do

```diff
@@ -0,0 +1,40 @@
+cap prog drop income_sim
+
+prog define income_sim, rclass
+
+clear
+
+syntax anything
+set obs `anything'
+
+gen state = 1+mod(_n,10)
+sort state
+gen id = _n
+gen age = rnormal(35,5)
+drop if age < 18
+gen educ_yrs = trunc(rnormal(12+(state/2),1))
+drop if educ_yrs < 0
+gen dist_from_city = rnormal(25-state,5)
+drop if dist_from_city < 0
+gen family_size = trunc(rnormal(4,1))
+drop if family_size < 1
+gen experience = runiform(0,age-18)
+
```

Comment:
Can you characterize here what your final sample size looks like? As in what share of your observations are lost?